.gitignore

```diff
@@ -3,5 +3,4 @@
 /lionweb-typescript.iml
 
 # NPM:
-/node_modules
-
+node_modules/
```

Comment:
This doesn't change the semantics from “just `node_modules` at root level” to “any `node_modules` directory”?

Reply:
`node_modules/` is fine, actually. Some packages will anyway have their own `node_modules/`.